front/package-lock.json

```diff
@@ -168,13 +168,15 @@
     },
     "../sdks/js": {
       "name": "@dust-tt/client",
-      "version": "1.0.41",
+      "version": "1.0.42",
```

Comment:
(Changes I had on main after pulling @aubin-tchoi's changes on the sdk - unrelated)